source/lib/JMSLab/tests/test_autofill.py

```diff
@@ -1,25 +1,64 @@
-import sys
-
-sys.path.append('./source/lib')
-
 from unittest import main, TestCase
-from JMSLab.autofill import GenerateAutofillMacros
+import tempfile, shutil 
 from os.path import exists
 
-
+from ..autofill import GenerateAutofillMacros
 
```

Comment:
Style-related: should we have all the `import stuff` together and the `from a import B` together, as in for instance [test_build_latex.py](https://github.com/JMSLab/Template/blob/db8ac17cbb45a65ec967d7bfdf5af94afc0c6329/source/lib/JMSLab/tests/test_build_latex.py)?